types/2020-08-27/PaymentIntents.d.ts

```diff
@@ -530,6 +532,23 @@ declare module 'stripe' {
           number: string | null;
         }
 
+        interface PaynowDisplayQrCode {
+          /**
+           * The raw data string used to generate QR code, it should be used together with QR code library.
+           */
+          data: string;
+
+          /**
+           * ​​The image_url_png string used to render QR code
```

Comment:
This is weird - for some reason there's two zero-width spacers between * and T.

```
const x = "* ​​T"
x.length => 5
x.charCodeAt(0) => 42 (*)
x.charCodeAt(1) => 32 (space)
x.charCodeAt(2) => 8203 (zero width spacer)
x.charCodeAt(3) => 8203 (zero width spacer)
x.charCodeAt(4) => 84 (T)
```

I think this is in the docstring itself in OpenAPI.